src/resources/js/app.js

```diff
@@ -374,13 +389,11 @@ const Charts = (() => {
 })();
 
 // For all charts, set miniumum Y to 0
-Chart.scaleService.updateScaleDefaults('linear', {
-    ticks: { suggestedMin: 0 }
-});
+Chart.defaults.scales.linear = { ticks: { suggestedMin: 0 } };
```

Comment:
I think `Chart.defaults.scale.linear.ticks.suggestedMin = 0;` is safer here

Reply:
I've checked in the "Run time changes" chart this was not working
the correct is `Chart.defaults.scales.linear.suggestedMin = 0`. Now working :)